pyproject.toml

```diff
@@ -568,3 +568,11 @@ pytest-cov = ">=6.1.1,<7"
 pytest-mock = ">=3.14.1,<4"
 pytest-pretty = ">=1.3.0,<2"
 pytest-pyvista = ">=0.2.0,<0.3"
+
+[tool.pixi.feature.test.tasks.test-clean]
+cmd = "rm -rf tests/plotting/image_cache test_images"
+
+[tool.pixi.feature.test.tasks.test]
+args = [{ "arg" = "option", "default" = "" }]
+cmd = "pytest {% if option | length %}-m '{{ option }}' {% endif %}--generated_image_dir test_images"
```

Comment:
`minijinja` ([0.46.0](https://github.com/prefix-dev/pixi/releases/tag/v0.46.0)) to the rescue 🚀